index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Create Mobile Version</title>
+</head>
+<body>
+    <main>
+        
+        <header>
+            <h1 class="logo">Welcome</h1>
+     
+            <nav class="menu-bar">
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+                <div class="bar"></div>
+            </nav>
+
+        </header>
+
+        <div class="background-image">
+          <div class="about-me">
+             
+                <p class="intro">Hey there. I’m Dozie <br><span class="who-am-i">I’m a software developer</span></p>
+                
+             <div>
```

Comment:
- [ Optional] Since `<p>` tag is a block element and you only have one paragraph here, you might not need to wrap it with a `<div>`.

Reply:
The div tag element has been removed to make the codes appear cleaner.